README.md

```diff
@@ -15,6 +15,7 @@ A collection of plugins for Jekyll:
 - Stickers Tag
 - Video Tag
 - Image Tag
+- Statistics Tag
```

Comment:
It this the Metrics Tag or is there another one called Statistics?

Reply:
Fixed the name :) Is the same one